python/tvm/relay/frontend/mxnet.py

```diff
@@ -367,7 +367,8 @@ def _mx_leaky_relu(inputs, attrs):
         return _op.nn.leaky_relu(inputs[0], alpha=attrs.get_float("slope", 0.25))
     if act_type == "prelu":
         assert len(inputs) == 2
-        return _op.nn.prelu(*inputs)
+        alpha = tvm.relay.multiply(tvm.relay.op.tensor.ones_like(inputs[0].tuple_value.args[-1]), inputs[1])
```

Comment:
Can we also add related test cases?

Reply:
@kevinthesun I didn't find the separated corresponding test file like what tf did. Could you indicate the location for these frontend op testing files?

Reply:
https://github.com/dmlc/tvm/blob/master/tests/python/frontend/mxnet/test_forward.py